internal/uvm/create.go

```diff
@@ -391,3 +393,42 @@ func (uvm *UtilityVM) CloseGCSConnection() (err error) {
 	}
 	return
 }
+
+// validateStorageSpaceEVDPath verifies the host path provided for the mount
+// type "ExtensibleVirtualDisk" and ExtensibleVirtualDiskType "space"
+// The correct path is in the "space://{pool_guid}{disk_guid}" format.
+func validateStorageSpaceEVDPath(path string) error {
+	if !strings.HasPrefix(path, `space://{`) ||
```

Comment:
can we do a regexp for this?

Reply:
I guess we could validate if the GUIDs are sound also with a regex seeming as how we base generating a `guid.GUID` type from the string we get below based off the assumption the user passed in a valid guid.

Reply:
I didn't do regex for 2 reasons:
1. This `path` is very specific to the storage space EVD path. The only characters that can change in this format are the GUIDs. Everything else must remain exactly the same. So we are not going to gain much by using a regex here.
2. For verifying the GUIDs we could use a regex but usually regex checks are costlier than just verifying if these 1 or 2 GUIDs are valid (as done by the `FromString` function).

Let me know what you guys think.

Reply:
Oh good call, I didn't even think about `FromString` just failing

Reply:
I'm fine with that then! If all we care about is if this ends with a "}" and begins with nine specific characters, this should be enough

Reply:
fine by me as well.